src/Components/Carouse/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import "./styles.css";
+
+import Opening from "../../assets/offers/opening.jpg";
+import Summer from "../../assets/offers/summer.png";
+import Toy from "../../assets/offers/toy.png";
+import Winter from "../../assets/offers/winter.png";
+
+function Carouse() {
+  var counter = 1;
```

Comment:
Don't use var.

Reply:
I believe there's a typo in the Carousel spelling

Reply:
Thanks Prabhav for pointing it out i have fixed it.